src/test/modules/test_misc/t/008_percona_server_version.pl

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/perl
+use strict;
+use warnings FATAL => 'all';
+use PostgreSQL::Test::Cluster;
+use PostgreSQL::Test::Utils;
+use Test::More;
+use lib 't';
+use Env;
+
+if (!defined($ENV{PERCONA_SERVER_VERSION}))
+{
+     plan skip_all => "PERCONA_SERVER_VERSION variable not define in the environment.";
```

Comment:
It's not clear what exactly does this test verify? It downloads one specific version and check version string, am I right? What kind of errors we can catch with such test?

Reply:
Yes, you are right! We need to update the variable in ci_scripts/env.sh (which will be present once this PR is merged), otherwise TAP case failure will let us know to update it. :)